Test CI is now failing without any merge into the repo

- First attempt last week: all green -> https://github.com/meilisearch/meilisearch-symfony/actions/runs/8233944664/attempts/1
- I re-ran the CI today: failing -> https://github.com/meilisearch/meilisearch-symfony/actions/runs/8233944664

```
 Script SYMFONY_DEPRECATIONS_HELPER='ignoreFile=./tests/baseline-ignore' ./vendor/bin/simple-phpunit --colors=always --verbose handling the test:unit event returned with error code 1
Error: Process completed with exit code 1.
```

Looks it's related to deprecation making tests failing, but I might be wrong. I'm not a symfony user.

Any contribution to remove/update them is welcome ❤️

--- a/.php-cs-fixer.dist.php
+++ b/.php-cs-fixer.dist.php
@@ -6,21 +6,17 @@ $finder = PhpCsFixer\Finder::create()
     ->in(__DIR__.'/src')
     ->in(__DIR__.'/tests')
     ->append([__FILE__]);
-$config = new PhpCsFixer\Config();
 
-$config->setRules([
-            '@Symfony' => true,
-            '@PHP80Migration:risky' => true,
-            'global_namespace_import' => [
-                'import_classes' => false,
-                'import_functions' => false,
-                'import_constants' => false,
-            ],
-            'no_superfluous_phpdoc_tags' => false,
-        ]
-)
+return (new PhpCsFixer\Config())
     ->setRiskyAllowed(true)
     ->setFinder($finder)
-;
-
-return $config;
+    ->setRules([
+        '@Symfony' => true,
+        '@PHP80Migration:risky' => true,
+        'global_namespace_import' => [
+            'import_classes' => false,
+            'import_functions' => false,
+            'import_constants' => false,
+        ],
+        'no_superfluous_phpdoc_tags' => false,
+    ]);

--- a/tests/config/config.yaml
+++ b/tests/config/config.yaml
@@ -25,3 +25,5 @@ doctrine:
                 dir: '%kernel.project_dir%/tests/Entity'
                 prefix: 'Meilisearch\Bundle\Tests\Entity'
                 alias: App
+        controller_resolver:
+            auto_mapping: false

--- a/tests/config/config_php7.yaml
+++ b/tests/config/config_php7.yaml
@@ -29,3 +29,5 @@ doctrine:
                 dir: '%kernel.project_dir%/tests/Entity'
                 prefix: 'Meilisearch\Bundle\Tests\Entity'
                 alias: App
+        controller_resolver:
+            auto_mapping: false

